docs/collections/keyvalue/key-value-store-data.md

```diff
@@ -3,3 +3,42 @@ title: Key-Value Store Data
 sidebar_position: 40
 ---
 
+The Key-Value Store Data tab is the primary section for viewing individual collection information.
+
+![Key-Value Store Data Tab](/img/collections/kv-store-data.png)
+
+This is a complex tab, which contains the following items.
```

Comment:
```suggestion
This is a complex tab which contains the following items.
```

Comma not necessary, but also not necessary to delete if we want it to flow better for non-native English speakers.